official/AnalisiDeiRequisiti/res/sections/requisiti.tex

```diff
@@ -121,7 +121,7 @@ \subsection{Requisiti funzionali}
 
 	%<< end
 
-    R1D 14 & Funzionale \newline Desiderabile & Il sistema permette all'utente di scaricare da \glossaryItem{Terminale} una specifica \glossaryItem{DSL}, in un formato leggibile dall'editor. &  Capitolato \newline  UC-E1 \newline \\ \hline
+    R1D 14 & Funzionale \newline Desiderabile & Il sistema permette all'utente di importare una definizione di DSL esistente scrivibile. &  Capitolato \newline  UC-E1 \newline \\ \hline
```

Comment:
In UC-E1 c'è scritto: L'utilizzatore può importare una specifica DSL esistente.
Quindi il requisiti è stato riscritto correttamente ora, in relazione a quel caso d'uso